src/scroll_list/ScrollList.js

```diff
@@ -1297,13 +1298,13 @@ define(function(require) {
          * Companion method to `scrollToItem` responsible for calculating and
          * applying the offset to the map panToOptions, depending on the viewportAnchorLocation
          */
-        _applyItemOffset: function(panToOptions, offset, itemScaleToFit, viewportAnchorLocation) {
+        _applyItemOffset: function(panToOptions, offset, itemScaleToFit, itemLayoutLeft, viewportAnchorLocation) {
```

Comment:
itemLayoutLeft isn't necessarily what it is, but it is a private function, so I'm not overly concerned.